src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -39,7 +71,7 @@ const useSaveAnonChatHook = () => {
     type: string
   ) => {
     if (!localDb) return;
-    const {channelName, channelImage, originalMembers} = await getChannelListInfo(
+    const {channelName, channelImage, originalMembers} = getChannelListInfo(
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getChannelListInfo` function was previously awaited, but the `await` keyword has been removed in this change. If `getChannelListInfo` is asynchronous and returns a Promise, this could lead to unexpected behavior. Please ensure that this function is not asynchronous or add back the `await` keyword if it is.

```diff
-    const {channelName, channelImage, originalMembers} = await getChannelListInfo(
+    const {channelName, channelImage, originalMembers} = getChannelListInfo(
       object,
       signedProfileId,
       anonProfileId
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->